docs/src/routes/Routes.js

```diff
@@ -87,7 +87,7 @@ import MapPage from '../pages/MapPage';
 import InfiniteScrollPage from '../pages/InfiniteScrollPage';
 import MobileScrollContainerPage from '../pages/MobileScrollContainerPage';
 import ModalsPage from '../pages/ModalsPage';
-import NativeFlatListPage from '../pages/NativeFlatListPage';
+import NativeFlatListPage from '../pages/FlatListPage';
```

Comment:
If we are renaming this should the import also be FlatListPage to be consistent?

Reply:
Yes